### PR TITLE
Debug api to dump all forkchoice proto nodes

### PR DESCRIPTION
**Pull request by twoeths**
**Motivation**

- We want to dump all proto nodes to know the current status of forkchoice to compare to other nodes/clients
- Other clients already did this (not part of the spec yet so make it `v0`)

**Description**

- Add `/eth/v0/debug/forkchoice` debug api to dump all proto nodes

### CODE REVIEW

**Comment by twoeths** [comment]
sample response from goerli 
[forkchoice_nodes.json.zip](https://github.com/ChainSafe/lodestar/files/10729270/forkchoice_nodes.json.zip)

File: packages/api/package.json [hunk 1]
```diff
@@ -74,6 +74,7 @@
     "@lodestar/params": "^1.4.3",
     "@lodestar/types": "^1.4.3",
     "@lodestar/utils": "^1.4.3",
+    "@lodestar/fork-choice": "1.4.3",
```
Comment (wemeetagain) [inline packages/api/package.json]: can you avoid this?
Maybe use ProtoNodeSszType instead?

Comment (twoeths) [reply packages/api/package.json]: thanks @wemeetagain , that works well 👍

**Review by wemeetagain — APPROVED** [review]
LGTM

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.6.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.6.0) :tada:

**Pull request merged**